lightning/src/events/bump_transaction.rs

```diff
@@ -11,6 +11,66 @@
 //!
 //! [`Event`]: crate::events::Event
 
+//! LDK offers the ability to its users to open Lightning [anchor outputs](https://bitcoinops.org/en/topics/anchor-outputs/)
+//! channels (`option_anchors_zero_fee_htlc_tx`) with its counterparties.
+//!
+//! Under the Lightning security model, confirmation of commitment and second-stage HTLC transactions
+//! (BOLT3's `HTLC-timeout` and `HTLC-preimage` and the corresponding spends on counterparty commitment
+//! transaction) must be realized in a time-sensitive manner if there are pending HTLCs. Lack of chain
+//! confirmation before HTLC expiration timelocks can constitute a loss of funds, as the counterparty can
+//! claim the HTLC output with a [competing transaction](https://bitcoinops.org/en/blog/waiting-for-confirmation/#policy-as-an-interface).
+//!
+//! Failure of confirmation might be provoked by blockspace demand spikes provoked by unrelated Bitcoin on-chain
+//! transaction issuers. Anchor output enables to remedy to this issue by allowing a Lightning node to dynamically
+//! adjust the feerate of its commitment and second-stage HTLC transactions in an unilateral way.
+//!
+//! However, anchor output channels are requiring from the user the provision and maintenance of fee-bumping
+//! reserves to attach feerate increase to the commitment (via a Child-Pay-For-Parent) or the second-stage HTLC
+//! transactions (via `SIGHASH_SINGLE | SIGHASH_ANYONECANPAY`). Those fee-bumping reserves should be a collection
+//! of UTXOs, of which the state and signing capabilites should be maintained on a "hot" host. A "hot" host is a
+//! computing machine with 24/7 Internet connectivity as fee-bumping UTXOs might need to be signed as soon there is
+//! a pending HTLC. A LDK user can implement management of fee-bumping reserves through the `CoinSelectionSource`
+//! and `WalletSource` traits.
+//!
+//! The total value of the collection of the UTXOs maintained must be high enough to pay for channel transactions
+//! weight at high-level of network mempools feerates.
+//!
+//! A _predictive_ worst-case `feerate_per_kw` based on historical network mempools traffic must be selected.
+//! Predictive worst-case must be understood as the worst-historical network mempools feerate with an additional
+//! safety margin. A factor of 2 is a conservative estimation.
+//!
+//! The number of opened anchor outputs channels must be selected, i.e `current_opened_channels`. As of 0.0.116
+//! release, LDK does not have a max number of opened channels enforced by default.
+//!
+//! The maximum number of HTLC outputs per-commitment transaction on both directions must be selected, i.e
+//! `max_htlc_allsides`.  As of 0.0.116 release, LDK has a BOLT3's `max_accepted_htlcs` of 50 (`DEFAULT_MAX_HTLCS`)
```

Comment:
See comment just above. In this context, this is used a convenience to yield the level of fee-bumping reserve that a LDK user is recommended to maintain. Specific helpers to compute it could be introduced.